client/src/components/login-form.js

```diff
@@ -1,106 +1,131 @@
-import React, { Component } from 'react'
-import { Redirect } from 'react-router-dom'
-import axios from 'axios'
+import React, { Component } from "react";
+import { Redirect } from "react-router-dom";
+import axios from "axios";
+import Button from "react-bootstrap/Button";
+import MyVerticallyCenteredModal from "./Modals/index";
 
 class LoginForm extends Component {
-    constructor() {
-        super()
-        this.state = {
-            email: '',
-            password: '',
-            redirectTo: null
-        }
-        this.handleSubmit = this.handleSubmit.bind(this)
-        this.handleChange = this.handleChange.bind(this)
-  
-    }
+  constructor() {
+    super();
+    this.state = {
+      modalShow: false,
+      email: "",
+      password: "",
+      redirectTo: null
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
 
-    handleChange(event) {
-        this.setState({
-            [event.target.name]: event.target.value
-        })
-    }
+  handleChange(event) {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  }
 
-    handleSubmit(event) {
-        event.preventDefault()
-        console.log('handleSubmit')
+  handleSubmit(event) {
+    event.preventDefault();
+    console.log("handleSubmit");
 
-        axios
-            .post('/user/login', {
-                email: this.state.email,
-                password: this.state.password
-            })
-            .then(response => {
-                console.log('login response: ')
-                console.log(response)
-                if (response.status === 200) {
-                    // update App.js state
-                    this.props.updateUser({
-                        loggedIn: true,
-                        email: response.data.email
-                    })
-                    // update the state to redirect to home
-                    this.setState({
-                        redirectTo: '/'
-                    })
-                }
-            }).catch(error => {
-                console.log('login error: ')
-                console.log(error);
-                
-            })
-    }
+    axios
+      .post("/user/login", {
+        email: this.state.email,
+        password: this.state.password
+      })
+      .then(response => {
+        console.log("login response: ");
+        console.log(response);
+        if (response.status === 200) {
+          // update App.js state
+          this.props.updateUser({
+            loggedIn: true,
+            email: response.data.email
+          });
+          // update the state to redirect to home
+          this.setState({
```

Comment:
you can make this 1 line since we are only changing 1 property it isnt hard to follow compared to like 2+ changing of states.